db create: reflect currently available versions

thbff opened the pull request:
* added postgresql 13 so that possibly new install won't end up with LC_ parameters unset
* as versions prior 8.4 are far beyond EOL, that part might be reworked a bit more "drastically" I guess (but have no idea what needs to be yet supported, so patching it this simple way)

franku commented on core/src/cats/create_bareos_database.in:
```diff
@@ -119,7 +119,7 @@ case ${db_type} in
       #
       case ${PSQLVERSION} in
-         9.*|10.*|11.*|12.*)
+         9.*|1[0123].*)
```
Could you replace all lines 117-131 by: 
`ENCODING="ENCODING 'SQL_ASCII' LC_COLLATE 'C' LC_CTYPE 'C'"`

thbff replied:
done, exactly what I meant by "more drastically"

thbff replied:
Looking at that again, it actually could/should be 105-131 as PSQLVERSION is not needed anymore

franku replied:
That's right. Good hint. Please remove this part, too.

franku reviewed (changes requested):
We would like to merge your PR. There are some things to be done, yet: 
- squash your commits to one and give it a reasonable description i.e. "scripts: cleanup code for postgresql db creation"
- add yourself to the AUTHORS list if you want
- add a short description to the CHANGELOG including the number of your PR using this syntax [PR #709], it will be automatically updated to a link when we create a release

Thanks for contribution!

thbff commented:
> We would like to merge your PR. There are some things to be done, yet:
> 
>     * squash your commits to one and give it a reasonable description i.e. "scripts: cleanup code for postgresql db creation"

think this is up to you as squash will happen during a merge, right?

>     * add yourself to the AUTHORS list if you want

actually, I just removed lines and did not add anything new ;-)


>     * add a short description to the CHANGELOG including the number of your PR using this syntax [PR #709], it will be automatically updated to a link when we create a release

added, should merge flawlessly with the current file

> Thanks for contribution!

my pleasure, thanks for the software!

franku commented:
1. We usually do not squash or reword commits in PR branches, therefore it is up to the author. If you need help, we can do.
2. AUTHORS file is a list of contributors. Therefore your name should be there.

We will be happy if you want to do that, would you?

thbff commented:
I see, ok, I was not aware of the workflow (nor the rebase squash possibility), that's my first one on github, had to try a few times aside before push. It should hopefully be ready now...

franku commented:
Good work, thank you!
